### ISSUE TITLE
wrong function

### ISSUE THREAD

**Issue by lodew920** [Ysz2022/NeRCo#8]
Did the code on lines 85 to 88 of the mlp.py file not work? I found that some code is commented, are these functions commented incorrectly？
There is something wrong with the function “def query_rgb(self, inp, coord, cell=None)“？

**Comment by Ysz2022** [comment]
Thanks for your interest in our work! :)

The commented part is the old version we have used in the previous experiments, we rewrite it with a simpler way, so the old version is commented. As far as I know, there is nothing wrong in “def query_rgb(self, inp, coord, cell=None)“. If you have any questions about it, please tell me and look forward to discussing with you!

**Comment by lodew920** [comment]
Thank you very much for your prompt reply
I found that this code is not used in the following code？
![Snipaste_2023-09-05_18-08-16](https://github.com/Ysz2022/NeRCo/assets/53033941/6c46a40c-1ec8-4611-ab87-331c7f854e16)
Besides, the subtraction of these two coordinates is equal to 0?
![image](https://github.com/Ysz2022/NeRCo/assets/53033941/bd101fe1-38af-4576-9135-68724eefa5a1)
Looking forward to your reply, maybe I misunderstood

**Comment by Ysz2022** [comment]
I'm very sorry😖, I fell ill a few days ago😰 and I only replied to your message now.

For your first question, admittedly, coord_ has not been used in our code. But we have used it in the previous version to get q_feat, as the commented part shows.

For your second question, our code is partly based on LIIF[1], a super-resolution method. In its code, "coord_" is the coordinate of high-resolution image and "q_coord" is the coordinate of low-resolution image, it uses "coord_" and "q_coord" to calculate the relative coordinate "rel_coord". Here as we do not need super-resolution, our "rel_coord" should be set as 0.

[5] Yinbo Chen, Sifei Liu, and Xiaolong Wang. Learning continuous image representation with local implicit image function. In Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition, 2021.

**Comment by Ysz2022** [comment]
I'm very sorry😖, I fell ill a few days ago😰 and I only replied to your message now.

For your first question, admittedly, coord_ has not been used in our code. But we have used it in the previous version to get q_feat, as the commented part shows.

For your second question, our code is partly based on LIIF[1], a super-resolution method. In its code, "coord_" is the coordinate of high-resolution image and "q_coord" is the coordinate of low-resolution image, it uses "coord_" and "q_coord" to calculate the relative coordinate "rel_coord". Here as we do not need super-resolution, our "rel_coord" should be set as 0.

[1] Yinbo Chen, Sifei Liu, and Xiaolong Wang. Learning continuous image representation with local implicit image function. In Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition, 2021.
